kas/anchor.go

```diff
@@ -66,6 +68,86 @@ func NewKASAnchor(kasConfig *KASConfig, db AnchorDB, bc BlockChain) *Anchor {
 	}
 }
 
+// AnchorPeriodicBlock periodically anchor blocks to KAS.
+// if given block is invalid, it does nothing.
+func (anchor *Anchor) AnchorPeriodicBlock(block *types.Block) {
+	if !anchor.kasConfig.Anchor {
+		return
+	}
+
+	if block == nil {
+		logger.Error("KAS Anchor : failed to get a block form the event")
+		return
+	}
+
+	if block.NumberU64()%anchor.kasConfig.AnchorPeriod != 0 {
+		return
+	}
+
+	anchor.anchorBlock(block)
```

Comment:
Doesn't it need to handle error?

Reply:
Yes, it is. If there is an error, `anchorBlock` will log.
In next stage, I will add more feature to retry anchoring in `anchorBlcok`.

Reply:
@KimKyungup Sorry for reopening the conversation, however, in that case I think `anchorBlock` should not return an error.

Reply:
But for API, the function should return an error.
API will be added in next PR.